apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/observe/api_collections/ApiEndpoints.jsx

```diff
@@ -300,7 +333,6 @@ function ApiEndpoints() {
         data['new'] = prettifyData.filter(x=> x.isNew)
         data['no_auth'] = prettifyData.filter(x => x.open)
         data['shadow'] = [ ...shadowApis ]
-        setLoading(false)
```

Comment:
where is this loading false, i can see loading true only

Reply:
at line 342, setLoading(false) was repeating so removed it